apis/externalsecrets/v1/secretstore_ngrok_types.go

```diff
@@ -0,0 +1,37 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1
+
+import (
+	esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
+)
+
+type NgrokProvider struct {
```

Comment:
I think it would be good to start writing links into persistent API Docs of ngrok here (https://ngrok.com/docs/api/#authentication has some nice docs from you folks, lets reuse this in the API docs)

Reply:
I restructured the auth a bit. Link added to the auth section in https://github.com/external-secrets/external-secrets/pull/5160/commits/005ca10e119a58798bc960c1e3dd1c0da4828760